config/environments/development.rb

```diff
@@ -36,7 +36,7 @@
   config.action_mailer.perform_caching = false
   config.action_mailer.delivery_method = :notify
   config.action_mailer.notify_settings = {
-    api_key: ENV.fetch('GOVUK_NOTIFY_API_KEY')
```

Comment:
Is this change necessary? The upside of `fetch` is that we won't be suprised by nil-errors.

Reply:
This is an awkward one because during the image build I you get an error about this key not being present in the image, which would be a security risk if it were present in the image.
The only way I could see for that not to raise an error during the build was to remove that fetch, but happy to change this if anyone can suggest an alternative.

Reply:
Having discussed this with @duncanjbrown  I will see if I can temporarily set a dummy value for the GOVUK_NOTIFY_API_KEY in the image generated in the Dockerfile and then successfully override that with an environment variable through docker-compose in the runtime container.
This seems to be the best way to work around this issue for now, as opposed to my previous solution of not using fetch.

Reply:
Setting a dummy value for the GOVUK_NOTIFY_API_KEY in the Dockerfile works.
Commit 258f6e9 addresses this.